README.rst

```diff
@@ -0,0 +1,39 @@
+=========
```

Comment:
Would rather this be a markdown file since most projects use markdown files for READMEs.